stake-pool/js/package.json

```diff
@@ -26,13 +28,20 @@
   "publishConfig": {
     "access": "public"
   },
+  "browser": "dist.browser/index.js",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "browserslist": [
+    "defaults",
+    "not IE 11"
+  ],
   "files": [
-    "/dist"
+    "/dist",
+    "/dist.browser"
   ],
   "license": "ISC",
   "dependencies": {
+    "@babel/runtime": "^7.16.5",
```

Comment:
We should be able to keep this in `devDependencies` since it's just for testing, if it's even needed at all!